.github/workflows/on-code-change.yml

```diff
@@ -32,10 +32,11 @@ jobs:
         password: ${{ secrets.DOCKERHUB_PASSWORD }}
     steps:
       - uses: actions/checkout@v4
-      - uses: jahia/jahia-modules-action/build@v2
+      # Version should not be hard-coded, but let's try this until the new release of the action
+      - uses: jahia/jahia-modules-action/build@02b57835b0e1dd77607bec5a0c3d837adbcb0957
         with:
           tests_module_type: 'npm'
-          tests_module_path: 'javascript-modules-engine/tests/jahia-module/'
+          tests_module_path: 'jahia-test-module/'
```

Comment:
No way to only move one level up, i.e. `tests_module_path: 'tests/jahia-test-module/'`?

Reply:
We can do everything as long as it not nested within `javascript-modules-engine/tests`. Would it be better to have `packages/` and `test-packages/` at the root?